sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExpression.scala

```diff
@@ -197,6 +198,43 @@ case class GpuWindowExpression(windowFunction: Expression, windowSpec: GpuWindow
     case other =>
       throw new IllegalStateException(s"${other.getClass} is not a supported window function")
   }
+
+  lazy val optimizedRunningWindow: Option[GpuRunningWindowFunction] = {
+    if (System.getProperty("TEST_RUNNING", "true").toBoolean &&
```

Comment:
It would be nice to have a comment here explaining why we would default to assuming a test is running and in what context someone would explicitly specify `-DTEST_RUNNING=false` to prevent this logic.  I don't see anything setting this property to use as a reference.

Reply:
Oops forgot to remove that. I was using that for performance testing without recompiling things.